Makefile

```diff
@@ -91,9 +91,12 @@ pull_translations: ## pull translations from Transifex
 push_translations: ## push source translation files (.po) from Transifex
 	tx push -s
 
+validate_translations: ## Test translation files
+	cd edx_proctoring && i18n_tool validate -v
```

Comment:
What does that -v option do? I don't see that documented here: https://github.com/edx/i18n-tools/blob/master/i18n/validate.py#L208

Reply:
@amangano-edx  it is from parent class https://github.com/edx/i18n-tools/blob/master/i18n/__init__.py#L25